### PR TITLE
added throttled

### PR DESCRIPTION
**Pull request by MalteKiefer**
Workaround for Intel throttling issues in Linux with Lenovo notebooks.

### CODE REVIEW

File: plugins/throttled.plugin/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"icon": "cpu",
+	"label": "throttled",
+	"description": "Workaround for Intel throttling issues in Linux with Lenovo notebooks.",
+	"license": "MIT",
+	"category": "Drivers",
```
Comment (kwizart) [inline plugins/throttled.plugin/metadata.json]: This is more about Tweaks I would say...

Comment (MalteKiefer) [reply plugins/throttled.plugin/metadata.json]: Maybe category devices?

Comment (kwizart) [reply plugins/throttled.plugin/metadata.json]: Well, Tweaks category isn't that full and it's really what this component does IMO.

Comment (MalteKiefer) [reply plugins/throttled.plugin/metadata.json]: Ok I will change and then I will merge

**Pull request merged**